api/types/load_traffic.go

```diff
@@ -304,3 +325,38 @@ func (m *KubeGroupVersionResource) Validate() error {
 	}
 	return nil
 }
+
+var PatchTypeMapping = map[string]string{
```

Comment:
Suggestion adding function instead of exported var.

```go
func GetCliJSONPatchType(name string) (string, bool) {
        switch name {
        case "json":
             return "application/json-patch+json", false
        ...
}
```

You can handle it in the following-up.